Packs/EDL/Integrations/EDL/README.md

```diff
@@ -99,6 +99,16 @@ Expected value is a string, supports newline characters (`\n`).
 
 When `PAN-OS: drop invalid URL entries` is enabled, any URL entry that is not compliant with PAN-OS URL format is dropped instead of rewritten.
 
+#### Exported Fields 
+For use with JSON and CSV formats - select specific Cortex XSOAR fields to export. If given the value 'all' - all Cortex XSOAR fields are exported. If empty - only value and type are exported.
+
+the optniol fields are:
+id - modified - sortValues - comments - indicator - value - source - sourceInstances - sourceBrands - investigationIDs - lastSeen - firstSeen - lastSeenEntryID - firstSeenEntryID - CustomFields - tags - expirationStatus - expirationSource - calculatedTime - lastReputationRun - modifiedTime - aggregatedReliability
+
+There are additional fields that do not appear here but are optional.
+And also your custom fields.
+You can try to do a search with 'all' in the 'Exported Fields' parameter and see which fields are returned. (not all fields there are optional for search)
```

Comment:
I think that if we can't mention the fields that cannot be searched then we better not mention this.
```suggestion
In addition to the system fields, you can also search for custom fields.
In order to get the list of all available fields to search by, you can configure the `Exported Fields` parameter with the `all` option and check the list returned. 
```